app/models/visit_group.rb

```diff
@@ -20,7 +20,8 @@ class VisitGroup < ActiveRecord::Base
   validates :arm_id,
             :name,
             presence: true
-  validates :day, presence: true, numericality: true
+  validates :day, presence: true, unless: "ENV.fetch('USE_EPIC'){nil} == 'false'"
```

Comment:
Wouldn't it make more sense to use:

```
unless: ENV.fetch('USE_EPIC'){false}
```

That would default to false and still work for those with USE_EPIC set to 'true'